stl/src/xmath.hpp

```diff
@@ -3,9 +3,7 @@
 
 #ifndef _XMATH
 #define _XMATH
-#include <errno.h>

```

Comment:
I believe that this should be restored as `<cerrno>` because `xxxdtent.hpp` includes only `xmath.hpp` and then uses `errno`.